### PR TITLE
fix: standardise and refactor comment headers

### PR DESCRIPTION
**Pull request by dgibbs64**
# Description

Some refactoring and standardising headers in modules. Contributors now point to the contribs page on github since multiple people contribute each module rather than just developer who first submitted the module. Also phasing out the term function in favour of module to describe the .sh files

Fixes #[issue]

## Type of change

* [ ] Bug fix (a change which fixes an issue).
* [ ] New feature (change which adds functionality).
* [ ] New Server (new server added).
* [x] Refactor (restructures existing code).
* [ ] Comment update (typo, spelling, explanation, examples, etc).

## Checklist

PR will not be merged until all steps are complete.

* [ ] This pull request links to an issue.
* [x] This pull request uses the `develop` branch as its base.
* [x] This pull request Subject follows the Conventional Commits standard.
* [x] This code follows the style guidelines of this project.
* [x] I have performed a self-review of my code.
* [x] I have checked that this code is commented where required.
* [x] I have provided a detailed with enough description of this PR.
* [x] I have checked If documentation needs updating.

## Documentation

If documentation does need updating either update it by creating a PR (preferred) or request a documentation update.
* User docs: https://github.com/GameServerManagers/LinuxGSM-Docs
* Dev docs: https://github.com/GameServerManagers/LinuxGSM-Dev-Docs

**Thank you for your Pull Request!**

### CODE REVIEW

File: lgsm/functions/alert_rocketchat.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
-# LinuxGSM alert_rocketchat.sh function
-# Author: Alasdair Haig
+# LinuxGSM alert_rocketchat.sh module
+# Author: Daniel Gibbs
```
Comment (brendenyule) [inline lgsm/functions/alert_rocketchat.sh:3]: This is probably not your intent, but they way you did this makes it look like you are claiming the merit of other devs.

Comment (dgibbs64) [reply lgsm/functions/alert_rocketchat.sh:3]: Quite the opposite I made the change to more fairly highlight contributors. 

I am the creator/lead developer of LinuxGSM so my name goes on every file as Author. There was a contributor line underneath that was used for contributors. Sometimes people got mixed up and used the Author instead of contributor.

So instead I decided to rely on, GitHub shows every person who has contributed to the code and does it automatically. 

What normally happened was the first person who created the module put their name in and subsequent contributors don't normally add their name. I felt it was unfair to other contributors as over the years some have contributed more code to a module than the original creator. Or some modules are simply a few lines of code with the person putting their name in and never contributing again. So to make things fair I instead linked to the GitHub contributors page that lists all contributors to LinuxGSM. It is also possible to see contributors to a specific module or using blame to see who wrote specific code within GitHub.

Basically, its to make sure contributors are displayed fairly and automatically. 

You are right to raise this concern and I hope this answers your question 😃

Comment (brendenyule) [reply lgsm/functions/alert_rocketchat.sh:3]: I kind of understand what you are saying, but these lines clearly indicate that you are the author of the alert_rocketchat.sh module, which it looks like you aren't. That's why I raised concern.
